src/errors.ts

```diff
@@ -0,0 +1,23 @@
+export abstract class HttpError extends Error {
+  public abstract get code(): number
+}
+
+export class ForbiddenError extends HttpError {
```

Comment:
Yay! was just mentioning to the @syedhasandigi about the Error class. I guess if you have not updated maynbe this is something you could use and/or add. I think it could be a good exercise after the story you are working on just to get a better grip on .ts

CC @syedhasandigi